client/packages/coldchain/src/Equipment/utils.ts

```diff
@@ -11,6 +11,7 @@ function baseAssetFields(t: TypedTFunction<LocaleKey>) {
   return [
     'id',
     t('label.asset-number'),
+    t('label.catalogue-item-code'),
```

Comment:
actual fixes are in this file - the order of these arrays matter, but they were in all sorts of orders so weren't mapping correctly 😁